src/robot.coffee

```diff
@@ -392,7 +392,6 @@ class Robot
   # Returns nothing.
   parseHelp: (path) ->
     @logger.debug "Parsing help for #{path}"
-    scriptName = Path.basename(path).replace /\.(coffee|js)$/, ''
```

Comment:
#917 actually depends on this being present. In addition, I'd much prefer to have test cases in place before modifying this code.